packages/runtime/container-runtime/api-report/container-runtime.legacy.alpha.api.md

```diff
@@ -37,7 +37,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents
     protected constructor(context: IContainerContext, registry: IFluidDataStoreRegistry, metadata: IContainerRuntimeMetadata | undefined, electedSummarizerData: ISerializedElection | undefined, chunks: [string, string[]][], dataStoreAliasMap: [string, string][], runtimeOptions: Readonly<Required<IContainerRuntimeOptions>>, containerScope: FluidObject, baseLogger: ITelemetryBaseLogger, existing: boolean, blobManagerSnapshot: IBlobManagerLoadInfo, _storage: IDocumentStorageService, createIdCompressor: () => Promise<IIdCompressor & IIdCompressorCore>, documentsSchemaController: DocumentsSchemaController, featureGatesForTelemetry: Record<string, boolean | number | undefined>, provideEntryPoint: (containerRuntime: IContainerRuntime) => Promise<FluidObject>, requestHandler?: ((request: IRequest, runtime: IContainerRuntime) => Promise<IResponse>) | undefined, summaryConfiguration?: ISummaryConfiguration);
     // (undocumented)
     protected addContainerStateToSummary(summaryTree: ISummaryTreeWithStats, fullTree: boolean, trackState: boolean, telemetryContext?: ITelemetryContext): void;
-    addedGCOutboundRoute(fromPath: string, toPath: string, messageTimestampMs?: number): void;
+    addedGCOutboundRoute(fromPath: string, toPath: string, messageTimestampMs?: number, fromPkg?: readonly string[], toPkg?: readonly string[]): void;
```

Comment:
The only place these may not be provided is when it's called by the data store context directly when it processes an attach message for a DDS. In that case, it should still have `fromPkg` because that's the data store itself. `toPkg` can be undefined in that case. 

However, this function is part of `IFluidParentContext` which is implemented also implemented by `FluidDataStoreContext` where `fromPkg` also needs to be optional.